compass/landice/tests/greenland/run_model.py

```diff
@@ -70,7 +70,9 @@ def __init__(self, test_case, velo_solver, name='run_model', subdir=None,
 
         # download and link the mesh
         self.add_input_file(filename='landice_grid.nc',
-                            target='gis20km.210608.nc', database='')
+                            url=‘https://web.lcrc.anl.gov/public/e3sm/
+                                 inputdata/glc/mpasli/mpas.gis20km/
+                                 gis_20km_r01.20210824.nc')
```

Comment:
```suggestion
                            url='https://web.lcrc.anl.gov/public/e3sm/
                                 inputdata/glc/mpasli/mpas.gis20km/
                                 gis_20km_r01.20210824.nc')
```
I think maybe a fancy quotation mark got put in at the beginning of the URL by mistake.  It seems to be confusing python.